### DISCUSSION TITLE
How can I add the parent checkbox to the Vuetify datagrid group table?

### DISCUSSION THREAD

**Discussion by eguvenc** [vuetifyjs/vuetify#15547] [General]
When I use the vuetify data datable group feature, the following view is obtained by default.

![vuetify-parent-checkbox](https://user-images.githubusercontent.com/482833/181934145-81279525-8490-4b21-911f-12af545d19af.png)

But I want to add a checkbox to grouped rows and add a parent checkbox to have child rows marked with a single selection

![vuetify-parent-checkbox2](https://user-images.githubusercontent.com/482833/181934153-831739ad-7b97-4f8a-80bd-1ce5c2ae682e.png)

The default example is available at this address. https://codepen.io/ersin-g-ven-/pen/PoREoNj

```
<div id="app">
  <v-app id="inspire">
    <h5>Selected: {{selected}}</h5>
    <v-data-table
      v-model="selected"
      :headers="headers"
      :items="desserts"
      item-key="name"
      show-select
      @click:row="handleClick"
      group-by="category"
      class="elevation-1"
      show-group-by
    >
     
    </v-data-table>
  </v-app>
</div><div id="app">
  <v-app id="inspire">
    <h5>Selected: {{selected}}</h5>
    <v-data-table
      v-model="selected"
      :headers="headers"
      :items="desserts"
      item-key="name"
      show-select
      @click:row="handleClick"
      class="elevation-1"
    >
     
    </v-data-table>
  </v-app>
</div>
```

**Comment by KaelWD** [comment]
You can use the group.header slot, the default implementation looks like this: https://codepen.io/kaelwd/pen/OJvMOOr?editors=1010

**Reply by eguvenc** [reply]
Hi, Thanks for your answer.
But,
When the top group checkbox is selected, I want the checkboxes below it to be selected as well.

How can i do this. Any idea ?

Here is my code.
https://codepen.io/ersin-g-ven-/pen/LYdeWor?editors=1010
